config/matty-go-cam.json

```diff
@@ -35,8 +35,9 @@
       "oak": {
         "driver": "osgar.drivers.oak_camera:OakCamera",
         "init": {
-          "fps": 10,
+          "fps": 20,
           "is_color": true,
+	  "video_encoder": "h264",
```

Comment:
here we should add also the H264 quality, but for that we need to merge "osgar/nn_mask" first